index.bs

```diff
@@ -2129,11 +2189,103 @@ an [=attribution trigger=] |trigger|:
     : [=aggregatable report/contributions=]
     :: The result of running [=create aggregatable contributions=] with |source| and |trigger|.
     : [=aggregatable report/serialized private state token=]
-    :: |trigger|'s [=attribution trigger/serialized private state token=]
+    :: null.
+    : [=aggregatable report/aggregation coordinator=]
+    :: |trigger|'s [=attribution trigger/aggregation coordinator=].
+    : [=aggregatable report/source registration time configuration=]
+    :: |trigger|'s [=attribution trigger/aggregatable source registration time configuration=].
+1. Return |report|.
+
+<h3 id="generating-randomized-null-reports">Generating randomized null reports</h3>
+
+To <dfn>obtain a randomized number of null reports</dfn> given a double |distributionParameter|:
+
+1. [=Assert=]: |distributionParameter| is between 0 and 1 ( both inclusive).
+1. Return a random non-negative integer from a geometric distribution with probability parameter
+    |distributionParameter|.
+
+To <dfn>obtain a null report</dfn> given an [=attribution trigger=] |trigger| and a [=moment=] |sourceTime|:
```

Comment:
Shall we assert that `|sourceTime|` is properly rounded?

Reply:
Do we want to round source time? Currently it's just trigger time + i days.

Reply:
Actually I think it's probably fine as is, the source time will be rounded during serialization. Wdyt?

Reply:
Oh right, that makes more sense :) I think this is fine then.